website/js/index.js

```diff
@@ -0,0 +1,70 @@
+const form = document.querySelector(".reviews-form");
+form.addEventListener("submit", (event) => {
+  event.preventDefault(); // Prevent the form from submitting in the traditional way
+
+  const textInput = document.querySelector(".review-text-input");
+  const ratingInput = document.querySelector(".review-rating-input");
+
+  const newReview = {
+    text: textInput.value,
+    rating: ratingInput.value,
+  };
+
+  reviews.push(newReview);
+  localStorage.setItem("reviews", JSON.stringify(reviews));
+
+  form.reset();
+
+  renderReviews();
+});
+
+const reviewsElement = document.querySelector(".reviews");
+
+function displayReview(review, index) {
+  const reviewTemplate = document.querySelector("[data-review-template]");
+  const reviewElement = reviewTemplate.content.cloneNode(true).children[0];
+
+  const reviewText = reviewElement.querySelector(".review-text");
+  const reviewRating = reviewElement.querySelector(".review-rating");
+
+  reviewText.textContent = review.text;
+  reviewRating.textContent = review.rating + "/5";
+
+  const deleteButton = reviewElement.querySelector(".review-delete-btn");
+  deleteButton.addEventListener("click", () => {
+    reviews.splice(index, 1);
+
+    localStorage.setItem("reviews", JSON.stringify(reviews));
+
+    renderReviews();
+  });
+
+  reviewsElement.appendChild(reviewElement);
+}
+
+function renderReviews() {
+  reviewsElement.innerHTML = "";
```

Comment:
Na dodavanje i brisanje review, svaki put se sve obriše, pa se ponovo rendera.
Može li se to malo bolje odradit? Neke od metoda možda za iskoristiti: append, prepend, remove.

Možemo i ovako mergeat, nije pogreška, pa možeš nekad ubuduće proučit ovaj problem. Ti odluči